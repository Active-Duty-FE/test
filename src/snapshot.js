export function generateConfig() {
  return {
    host: "localhost",
    port: "3000",
  };
}

export function generateOhterConfig() {
  return {
    host: "localhost",
    port: "30001ww",
    time: new Date(),
  };
}
