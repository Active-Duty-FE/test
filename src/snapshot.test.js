import { generateConfig, generateOhterConfig } from "./snapshot";

test("test config", () => {
  expect(generateConfig()).toMatchSnapshot();
});

test("test config", () => {
  expect(generateOhterConfig()).toMatchInlineSnapshot({
  time: expect.any(Date)
}, `
{
  "host": "localhost",
  "port": "30001ww",
  "time": Any<Date>,
}
`);
});
