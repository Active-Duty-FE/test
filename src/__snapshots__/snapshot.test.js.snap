// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test config 1`] = `
{
  "host": "localhost",
  "port": "3000",
}
`;

exports[`test config 2`] = `
{
  "host": "localhost",
  "port": "30001ww",
  "time": Any<Date>,
}
`;
